workspaces/scanner/src/index.ts

```diff
@@ -8,6 +8,7 @@ import pacote from "pacote";
 import { getLocalRegistryURL } from "@nodesecure/npm-registry-sdk";
 import * as tarball from "@nodesecure/tarball";
 import type { PackageJSON } from "@nodesecure/npm-types";
+import Config from "@npmcli/config";
```

Comment:
```suggestion
import type Config from "@npmcli/config";
```

We never use the Config class in runtime so I guess we can use it as type only (same on depWalker.ts)